src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -288,7 +293,7 @@ const ShareInviteDialog = (props: ShareInviteDialogProps): JSX.Element => {
               <BaseCheckbox checked={notifyUser} />
               <p className="ml-2 text-base font-medium">{translate('modals.shareModal.invite.notifyUsers')}</p>
             </div>
-            <Button variant="primary" onClick={onInvite} disabled={isInviteButtonDisabled}>
+            <Button variant="primary" onClick={onInvite} disabled={isInviteButtonDisabled} loading={isAnyInviteLoading}>
```

Comment:
I think we should deactivate the button while the invitation is being processed.

Reply:
It disables if "loading" or "disabled" is true
![image](https://github.com/internxt/drive-web/assets/143480783/e5ff37f2-96b8-4745-93a4-e8da16edb20e)